index.html

```diff
@@ -63,6 +65,196 @@ <h1 class="grettings">I’m Kender Glad to see you!</h1>
+            <li>javascript</li>
+          </ul>
+
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          class="project-screenshot"
+          src="./assets/multi-post-stories.png"
+          alt="project image"
+        />
+
+        <div class="project-desc">
+          <h2>Multi-Post Stories</h2>
+          <ul class="s-desc">
+            <li>CANOPY</li>
+            <li><div class="bullet"></div></li>
+            <li>Bac End Dev</li>
+            <li><div class="bullet"></div></li>
+            <li>2015</li>
+          </ul>
+          <p class="desc-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          class="project-screenshot"
+          src="./assets/tonic-2.png"
+          alt="project image"
+        />
+
+        <div class="project-desc">
+          <h2>Tonic</h2>
+          <ul class="s-desc">
+            <li>CANOPY</li>
+            <li><div class="bullet"></div></li>
+            <li>Bac End Dev</li>
+            <li><div class="bullet"></div></li>
+            <li>2015</li>
+          </ul>
+          <p class="desc-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="card-works">
+        <img
+          class="project-screenshot"
+          src="./assets/multi-post-stories-2.png"
+          alt="project image"
+        />
+
+        <div class="project-desc">
+          <h2>Multi-Post Stories</h2>
+          <ul class="s-desc">
+            <li>CANOPY</li>
+            <li><div class="bullet"></div></li>
+            <li>Bac End Dev</li>
+            <li><div class="bullet"></div></li>
+            <li>2015</li>
+          </ul>
+          <p class="desc-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!-- ABOUT SECTION -->
+    <section class="about-section">
+      <div class="about-content">
+        <h1 class="about-title">About Myself</h1>
```

Comment:
Kindly, I suggest you only put 1 H1 header on your web page and then follow the sequence H2, H3, etc. Depending on the importance, this action will greatly help the accessibility of your website.

Reply:
Thank you for your review. I will remove it and replace it by an H2 tag.